cloudfoundry-client/src/main/lombok/org/cloudfoundry/uaa/identityzonemanagement/CreateIdentityZoneRequest.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright 2013-2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.cloudfoundry.uaa.identityzonemanagement;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+import lombok.Data;
+import lombok.Getter;
+import org.cloudfoundry.Validatable;
+import org.cloudfoundry.ValidationResult;
+
+import static com.fasterxml.jackson.annotation.JsonInclude.Include.NON_NULL;
+
+/**
+ * The request payload for the create identity zone operation
+ */
+@Data
+public final class CreateIdentityZoneRequest implements Validatable {
+
+    /**
+     * The description of the identity zone.
+     *
+     * @param description the description
+     * @return the description
+     */
+    @Getter(onMethod = @__(@JsonProperty("description")))
+    private final String description;
+
+    /**
+     * The id of the identity zone. When not provided, an identifier will be generated
+     *
+     * @param identityZoneId the identity zone id
+     * @return the identity zone id
+     */
+    @Getter(onMethod = @__({@JsonProperty("id"), @JsonInclude(NON_NULL)}))
```

Comment:
In general we don't need the `@JsonInclude(NON_NULL)` annotation.  The `ObjectMapper` used by the `RestTemplate` is [configured for that](https://github.com/cloudfoundry/cf-java-client/blob/master/cloudfoundry-client-spring/src/main/lombok/org/cloudfoundry/spring/util/network/OAuth2RestTemplateBuilder.java#L121), for all properties.